diff --git a/node_modules/eslint-find-rules/dist/lib/rule-finder.js b/node_modules/eslint-find-rules/dist/lib/rule-finder.js
index ac57728..c032865 100644
--- a/node_modules/eslint-find-rules/dist/lib/rule-finder.js
+++ b/node_modules/eslint-find-rules/dist/lib/rule-finder.js
@@ -117,6 +117,12 @@ function _getPluginRules(config) {
   /* istanbul ignore else */
   if (plugins) {
     plugins.forEach(function (plugin) {
+      // Skip checking current plugin for deprecated rules
+      // There is an issue when you to load itself with 'require'
+      if (plugin === 'deprecated-rules') {
+        return;
+      }
+
       var normalized = normalizePluginName(plugin);
       var pluginConfig = require(normalized.module);
       var rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules;
diff --git a/node_modules/eslint-find-rules/src/lib/rule-finder.js b/node_modules/eslint-find-rules/src/lib/rule-finder.js
index 3f8d19a..7e961f1 100644
--- a/node_modules/eslint-find-rules/src/lib/rule-finder.js
+++ b/node_modules/eslint-find-rules/src/lib/rule-finder.js
@@ -58,6 +58,12 @@ function _getPluginRules(config) {
   /* istanbul ignore else */
   if (plugins) {
     plugins.forEach(plugin => {
+      // Skip checking current plugin for deprecated rules
+      // There is an issue when you to load itself with 'require'
+      if (plugin === 'deprecated-rules') {
+        return;
+      }
+
       const normalized = normalizePluginName(plugin);
       const pluginConfig = require(normalized.module);
       const rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules;
